terroroftinytown/tracker/base.py

```diff
@@ -16,10 +16,11 @@ def get_current_user(self):
         if username_raw and token:
             username = username_raw.decode('ascii')
 
-            if username:
-                return User.check_account_session(username, token)
+            if username and User.check_account_session(username, token):
+                return username
 
     def prepare(self):
+        self.application.log_filter.username = self.get_current_user()
```

Comment:
This doesn't seem like a good idea since  global state is being changed and the username will likely be wrong once it leaves this function. It might be better to have a function defined on the base request handler that calls the logging with the username.